### PR TITLE
H0 tuning and layout of CREST Keywords

### PR DESCRIPTION
**Pull request by gasevic**
I added H0 tuning to the singlepoints tab and changed the layout for CREST keywords.

### CODE REVIEW

File: source/sp.rst [hunk 1]
```diff
@@ -485,3 +485,42 @@ Example: BF\ :sub:`3`\
...
+
+H\ :sub:`0`\  Tuning
+====================
+
+For special cases it can be beneficial to tune the H\ :sub:`0`\  Hamiltonian by modifying the atom-pairwise parameters. In order to do this, atom pairs need to be added to the ``$pairpar`` block in the parameter file ``$XTBPATH/param_gfn2-xtb.txt``.
+
+
+**Example entry for a nitrogen-tellurium pair**
+
+.. code:: bash
```
Comment (awvwgk) [inline source/sp.rst:497]:
```suggestion
.. code-block:: bash
    :caption: Example entry for a nitrogen-tellurium pair
```

File: source/sp.rst [hunk 2]
```diff
@@ -485,3 +485,42 @@ Example: BF\ :sub:`3`\
...
+   7  52  0.93
+ $end
+
+The first two numbers express atomic numbers followed by the H\ :sub:`0`\ -scaling factor. The default value for the scaling of each pair is 1.00 but certain pairs may already be changed in the file.
+
+Alternatively, an external parameter file can be imported with the commandline call:
+
+.. code:: sh
+
+  > xtb coord --vparam <PARAMETER_FILE>
```
Comment (awvwgk) [inline source/sp.rst:509]:
```suggestion
   xtb coord --vparam <PARAMETER_FILE>
```

File: source/sp.rst [hunk 3]
```diff
@@ -485,3 +485,42 @@ Example: BF\ :sub:`3`\
       4 F    -0.233   -0.335   -0.284
 
 The Fukui indexes for BF\ :sub:`3`\  indicate the most negative f(+) value and a positive value for f(-) at the boron atom. Thus, a nucleophilic attack can be expected at the boron atom.
+
+H\ :sub:`0`\  Tuning
+====================
+
+For special cases it can be beneficial to tune the H\ :sub:`0`\  Hamiltonian by modifying the atom-pairwise parameters. In order to do this, atom pairs need to be added to the ``$pairpar`` block in the parameter file ``$XTBPATH/param_gfn2-xtb.txt``.
```
Comment (awvwgk) [inline source/sp.rst:492]: Should we really recommend to modify the global GFN2-xTB parameter path here? I would prefer that we encourage the creation of a copy with a meaningful name that is than further modified.

**Review by awvwgk — COMMENTED** [review]
@pprcht Could you have a look on the crest command line documentation changes.

**Review by pprcht — APPROVED** [review]

**Review by awvwgk — APPROVED** [review]

**Pull request merged**